WcaOnRails/app/jobs/application_job.rb

```diff
@@ -1,4 +1,19 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  class DeferJob < StandardError
+    attr_reader :wait_time
+    def initialize(wait_time)
+      super("Delay the job by #{time} s")
```

Comment:
Oh, I completely missed that. Don't think I've seen `rescue_from` before.

So this violates the "don't use exceptions for regular control flow" rule, and if there is a simple way to do it "normally", that would be nice. But I can't think of one right away, and a simple comment saying "this is always caught in the rescue_from below" would be fine.

Also, if the string is never displayed, why pass one in to begin with?